hack/clean_vendor.sh

```diff
@@ -1,4 +1,8 @@
+dep ensure &>/dev/null
 find vendor \( -type f -or -type l \)  -not -name "*.go" -not -name "LICENSE" -not -name "*.s" -not -name "PATENTS" -not -name "*.h" -not -name "*.c" | xargs -I {} rm {}
 # delete all test files
 find vendor -type f -name "*_generated.go" | xargs -I {} rm {}
 find vendor -type f -name "*_test.go" | xargs -I {} rm {}
+find vendor -type f -name "AUTHORS" -name "AUTHORS.md" -iname 'CONTRIBUTORS' -iname 'NOTICE' | xargs -I {} rm {}
```

Comment:
This is duplicated. 
`find vendor \( -type f -or -type l \)  -not -name "*.go" -not -name "LICENSE" -not -name "*.s" -not -name "PATENTS" -not -name "*.h" -not -name "*.c" | xargs -I {} rm {}` will delete all file expect "*.go", "*.s", "PATENTS" ,and "LICENSE"".